scripts/build.ps1

```diff
@@ -54,9 +54,13 @@ if ($Platform -like "?64") {
     $gnargs += ' v8_enable_pointer_compression=true'
 }
 
+# TODO: N-API implementation of external ArrayBuffer doesn't comply with the Sandbox requirements for all memory allocations to be owned by the VM
+$gnargs += ' v8_enable_sandbox=false'

```

Comment:
>$gnargs [](http://example.com/codeflow?start=0&length=7)

In the latest Node-API this issue is addressed by conditional compilation for such cases. 
We should bring the latest code to address it.